test/features.js

```diff
@@ -0,0 +1,258 @@
+import {setup, teardown, showDocument, shouldThrowError} from './utils';
```

Comment:
part 2: porting the python tests. Most of the infrastructure is already in place as you can see (describe blocks). Look at ``http.js`` for examples. Most of the tests might seem overkill because coverage is almost 100% complete already, but they actually aren't because they're also the tests of subprojects like ``couchdb-render``.